src/run.ts

```diff
@@ -26,6 +26,40 @@ import type {} from "@octokit/plugin-throttling/dist-types/types.d";
 // To avoid that, we ensure to cap the message to 60k chars.
 const MAX_CHARACTERS_PER_MESSAGE = 60000;
 
+const setupOctokit = (githubToken) => {
```

Comment:
oh, now it makes sense... I totally missed the fact that each command~ has its own octokit instance

Reply:
Same. Want me to refactor so we have a single instance for the entire module ?